### ISSUE TITLE
Provide option of single page

### ISSUE THREAD

**Issue by rdelpeso** [kennethreitz/python-guide#708]
Hi, while I understand that this guide is a living document, it would be really awesome if there was a way to get all of its content as a single page on its current state. Don't know how achievable that would be (I will do a bit of research on free time and contribute if find anything).

This would allow for an easy download of the guide to read outside the PC. Say using a kindle for example.

Thoughts?

**Comment by kennethreitz** [comment]
There should be a PDF version available to you in the floating drop-down menu. Would that suit your needs?

Perhaps that should be more prominently linked-to.

**Comment by rdelpeso** [comment]
Ignore me... apparently requesting features while half asleep is not a good idea... I just noticed the floating drop-down you mention and that one contains this:

    Downloads
    PDF | HTML | Epub

Which is everything i ever wanted and more. Love it! Closing this request as user blindness :+1:

Thanks a lot for all the work that's been put into this guide. It has helped me lots and continues to do so.
